src/ResourceManager/Automation/Commands.Automation/Microsoft.Azure.Commands.ResourceManager.Automation.dll-help.xml

```diff
@@ -14064,4 +14064,204 @@ PS C:\&gt; Start-AzureRmAutomationDscCompilationJob -ConfigurationName "Config01
     </maml:relatedLinks>
   </command:command>
 
+  <command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
+    <command:details>
+      <command:name>Get-AzureRMAutomationHybridWorkerGroupGroup</command:name>
```

Comment:
There is an extra "Group" in the command name, causing the build to fail with "missing help" for the cmdlet.